pkg/api/defaults.go

```diff
@@ -563,6 +563,9 @@ func (p *Properties) setVMSSDefaultsForMasters() {
 	if p.MasterProfile.HasAvailabilityZones() && (p.OrchestratorProfile.KubernetesConfig != nil && p.OrchestratorProfile.KubernetesConfig.LoadBalancerSku == "") {
 		p.OrchestratorProfile.KubernetesConfig.LoadBalancerSku = StandardLoadBalancerSku
 		p.OrchestratorProfile.KubernetesConfig.ExcludeMasterFromStandardLB = to.BoolPtr(DefaultExcludeMasterFromStandardLB)
+		if p.OrchestratorProfile.KubernetesConfig.OutboundRuleIdleTimeoutInMinutes == 0 {
```

Comment:
This defaults code flow is only entered in when the master node(s) is/are VMSS. Is that what we want?

Reply:
I suppose because we're doing it in the `setVMSSDefaultsForAgents` flow below, we're covering both vectors. Maybe my question is: is this a VMSS-specific default?

Reply:
It is the default value when a standard load balancer is used. Since VMSS with availability zone uses a standard load balancer, I just reuse the same value.